@fabaff fixed.

Does anybody know how I can exclude certain unittests? alot uses `python setup.py test` with the standard lib unitests module from python.  There are exactly two tests that fail in NixOS' sandbox:
```
======================================================================
FAIL: test_no_spawn_no_stdin_attached (tests.commands.test_global.TestExternalCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/utilities.py", line 189, in _actual
    return loop.run_until_complete(coro(*args, **kwargs))
  File "/nix/store/i1m8r7mv8h47wr850cdsxksy22lv6gsz-python3-3.9.6/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/build/source/tests/commands/test_global.py", line 129, in test_no_spawn_no_stdin_attached
    ui.notify.assert_not_called()
  File "/nix/store/i1m8r7mv8h47wr850cdsxksy22lv6gsz-python3-3.9.6/lib/python3.9/unittest/mock.py", line 868, in assert_not_called
    raise AssertionError(msg)
AssertionError: Expected 'notify' to not have been called. Called 1 times.
Calls: [call('editor has exited with error code 1 -- No stderr output', priority='error')].

======================================================================
FAIL: test_env_set (tests.test_helper.TestCallCmdAsync)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/utilities.py", line 189, in _actual
    return loop.run_until_complete(coro(*args, **kwargs))
  File "/nix/store/i1m8r7mv8h47wr850cdsxksy22lv6gsz-python3-3.9.6/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/build/source/tests/test_helper.py", line 433, in test_env_set
    self.assertEqual(ret[0], 'bar')
AssertionError: '' != 'bar'
+ bar
```
 and because of this we have `doCheck=false` in the alot derivation.  I would like to deactivate these two and run the remaining tests.

--- a/pkgs/development/python-modules/alot/default.nix
+++ b/pkgs/development/python-modules/alot/default.nix
@@ -1,4 +1,4 @@
-{ lib, buildPythonPackage, python, fetchFromGitHub, isPy3k
+{ lib, buildPythonPackage, python, fetchFromGitHub, isPy3k, pytestCheckHook
 , notmuch2, urwid, urwidtrees, twisted, python_magic, configobj, mock, file, gpgme
 , service-identity, gnupg, sphinx, gawk, procps, future , withManpage ? false
 }:
@@ -35,11 +35,14 @@ buildPythonPackage rec {
     gpgme
   ];
 
-  # some twisted tests need the network (test_env_set... )
-  doCheck = true;
   postBuild = lib.optionalString withManpage "make -C docs man";
 
-  checkInputs =  [ gawk future mock gnupg procps ];
+  checkInputs = [ gawk future mock gnupg procps pytestCheckHook ];
+  # some twisted tests need internet access
+  disabledTests = [
+    "test_env_set"
+    "test_no_spawn_no_stdin_attached"
+  ];
 
   postInstall = let
     completionPython = python.withPackages (ps: [ ps.configobj ]);

